runtime/src/lib.rs

```diff
@@ -503,6 +512,29 @@ pub type ReferendumInstance = referendum::Instance1;
 pub type ReferendumModule = referendum::Module<Runtime, ReferendumInstance>;
 pub type CouncilModule = council::Module<Runtime>;
 
+// Production coucil and elections configuration
+#[cfg(not(any(feature = "staging_runtime", feature = "testing_runtime")))]
+parameter_types! {
+    // referendum parameters
+    pub const MaxSaltLength: u64 = 32;
+    pub const VoteStageDuration: BlockNumber = 14400;
+    pub const RevealStageDuration: BlockNumber = 14400;
+    pub const MinimumVotingStake: u64 = 10000;
```

Comment:
Not sure about the mismatch between `MinimumVotingStake` and `MinCandidateStake`, but let's avoid fiddling more than necessary now!

Reply:
I'll revert to original values since we tested with these values.